shared/profile/edit-profile/index.js

```diff
@@ -1,65 +1,33 @@
 // @flow
-import React, {Component} from 'react'
 import Render from './render'
+import {compose, withHandlers, withPropsOnChange, withState} from 'recompose'
 import {connect} from 'react-redux'
 import {editProfile} from '../../actions/profile'
 import {maxProfileBioChars} from '../../constants/profile'
+import type {TypedState} from '../../constants/reducer'
 import {navigateUp} from '../../actions/route-tree'
 
-import type {Props} from './render'
-
-class EditProfile extends Component<void, Props, void> {
-  render() {
-    const bioMaxChars = maxProfileBioChars
-    const bioLengthLeft = bioMaxChars - this.props.bio.length
-    return (
-      <Render
-        bio={this.props.bio}
-        bioLengthLeft={bioLengthLeft}
-        fullname={this.props.fullname}
-        location={this.props.location}
-        onBack={this.props.onBack}
-        onBioChange={this.props.onBioChange}
-        onCancel={this.props.onBack}
-        onFullnameChange={this.props.onFullnameChange}
-        onLocationChange={this.props.onLocationChange}
-        onSubmit={this.props.onEditProfile}
-      />
-    )
-  }
+const mapStateToProps = (state: TypedState) => {
+  // $FlowIssue
```

Comment:
just curious what the flowissue is here, username might be null? if so you can do a throw and it shouldn't need a flowissue